api.md

```diff
@@ -365,52 +402,6 @@ PostGIS geospatial indexes).  During testing, you might check your
 total chunk sizes via the [`chunk_relation_size`](#chunk_relation_size)
 function.
 
-**Space partitions**: The use of additional partitioning is a very
```

Comment:
Should there at least be a mention of the parameter, since it is still in the API parameters list?  Leave a warning about using it?

Reply:
Ok - added a really small note back in.